.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+    "extends": "airbnb",
+    "parser": "babel-eslint",
+    "parserOptions": {
+        "ecmaFeatures": {
+            "experimentalObjectRestSpread": true,
+            "jsx": true,
+            "es6": true
+        },
+        "sourceType": "module"
+    },
+    "plugins": [
+        "babel",
+        "react"
+    ],
+    "rules": {
+        "indent": ["error", 4],
+        "linebreak-style": [ "error", "unix" ],
+        "quotes": [ "error", "single" ],
+        "semi": [ "error", "always"],
+        "react/jsx-boolean-value": 0,
+        "react/jsx-closing-bracket-location": 1,
+        "react/jsx-curly-spacing": [2, "always"],
+        "react/jsx-indent-props": [1, 2],
+        "react/jsx-no-undef": 1,
+        "react/jsx-uses-react": 1,
+        "react/jsx-uses-vars": 1,
+        "react/wrap-multilines": 1,
+        "react/react-in-jsx-scope": 1,
+        "react/prefer-es6-class": 1,
+        "react/jsx-no-bind": 1
```

Comment:
do we need to specify so many configs? I thought Airbnb would do that for us.

Reply:
We don't but I've been meaning to ask you whether we want to tweak or change some of them

Reply:
nope. I would rather go with standard airbnb and then tweak I want to do it differently